translations/strings.yaml

```diff
@@ -39,3 +39,5 @@ Universal frequently used notation: ''
 universal glossary: ''
 Vocabulary: ''
 What it means: ''
+Input: ''
+Select a circuit in the selector above: ''
```

Comment:
It would be nice to order them alphabetically.

Reply:
All the strings? I think that it is not worth it. If we change a copy, we will need to reorder it. It is mostly "ordered" by components.